docs/Toy/toy/emulator/toy_accelerator_instruction_functions.py

```diff
@@ -3,19 +3,15 @@
 from xdsl.dialects import riscv
 from xdsl.interpreter import Interpreter, PythonValues, impl, register_impls
 from xdsl.interpreters.riscv import RawPtr, RiscvFunctions
-from xdsl.interpreters.shaped_array import ShapedArray
 
 
 @register_impls
 class ToyAcceleratorInstructionFunctions(RiscvFunctions):
     def __init__(self):
         super().__init__(
             custom_instructions={
-                "tensor.print1d": accelerator_tensor_print1d,
-                "tensor.print2d": accelerator_tensor_print2d,
                 "buffer.alloc": accelerator_buffer_alloc,
                 "buffer.copy": accelerator_buffer_copy,
```

Comment:
I don't remember exactly what the blocker was to remove these functions, but would love to do that also with @AntonLydike's help

Reply:
That seems interesting, can you create an issue and assign it to me? That way I hopefully don't forget!